apps/wallet-dashboard/hooks/useGetSupplyIncreaseVestingObjects.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetCurrentEpochStartTimestamp } from '@/hooks';
+import {
+    SupplyIncreaseVestingPayout,
+    SupplyIncreaseVestingPortfolio,
+    TimelockedObject,
+    VestingOverview,
+} from '@/lib/interfaces';
+import {
+    buildSupplyIncreaseVestingSchedule,
+    ExtendedDelegatedTimelockedStake,
+    formatDelegatedTimelockedStake,
+    getLatestOrEarliestSupplyIncreaseVestingPayout,
+    getVestingOverview,
+    isSupplyIncreaseVestingObject,
+    isTimelockedUnlockable,
+    mapTimelockObjects,
+} from '@/lib/utils';
+import {
+    TIMELOCK_IOTA_TYPE,
+    useGetAllOwnedObjects,
+    useGetTimelockedStakedObjects,
+    useUnlockTimelockedObjectsTransaction,
+} from '@iota/core';
+import { Transaction } from '@iota/iota-sdk/transactions';
+
+export function useGetSupplyIncreaseVestingObjects(address: string): {
+    nextPayout: SupplyIncreaseVestingPayout | undefined;
+    lastPayout: SupplyIncreaseVestingPayout | undefined;
+    vestingSchedule: VestingOverview;
+    vestingPortfolio: SupplyIncreaseVestingPortfolio | undefined;
+    vestingMapped: TimelockedObject[];
+    vestingStakedMapped: ExtendedDelegatedTimelockedStake[];
+    unlockAllTimelockedObjects:
+        | {
+              transactionBlock: Transaction;
+          }
+        | undefined;
+} {
+    const { data: currentEpochMs } = useGetCurrentEpochStartTimestamp();
+
+    const { data: timelockedObjects } = useGetAllOwnedObjects(address || '', {
+        StructType: TIMELOCK_IOTA_TYPE,
+    });
+    const { data: timelockedStakedObjects } = useGetTimelockedStakedObjects(address || '');
+
+    const vestingMapped = mapTimelockObjects(timelockedObjects || []).filter(
```

Comment:
I don't know if it's possible, but for me, it's confusing that in some places we use `vesting`, in others `supply increase`, and in others `timelocked`, can we unify all these terms? I mean, use one of them and apply it to all the related variables? :pray:

Reply:
`vesting` and `supplyIncrease` are the same, but not `timelocked`. We can unify the first two okay?